src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,22 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
```

Comment:
Verify that `AuthenticationServiceImpl` correctly implements the `AuthenticationService` interface and that it is properly configured to handle login operations.